lua/nvim-tundra/base_configuration.lua

```diff
@@ -32,6 +32,7 @@ local base_configuration = {
   plugins = {
     lsp = true,
     treesitter = true,
+    nvimtree = true,
```

Comment:
This should be false by default. I think only 'base' plugins such as native LSP and Treesitter should be enabled by default.

```suggestion
    nvimtree = false,
```